src/store/slices/solanaSlice.ts

```diff
@@ -158,49 +126,25 @@ export const makePayment = createAsyncThunk(
 export const makeCollectablePayment = createAsyncThunk(
   'solana/makeCollectablePayment',
   async (
-    {
-      account,
-      collectable,
-      payee,
-      cluster,
-      anchorProvider,
-    }: CollectablePaymentInput,
+    { account, transferTxn, cluster, anchorProvider }: CollectablePaymentInput,
     { dispatch },
   ) => {
     if (!account?.solanaAddress) throw new Error('No solana account found')
 
-    const compressedNFT = collectable as CompressedNFT
-    const nft = collectable as Collectable
-
     try {
-      const transfer = compressedNFT?.compression?.compressed
```

Comment:
Why did all of this get removed?

Reply:
Transaction creation got moved to the `useSubmitTxn` hook in order to simulate it with the bottom sheet